src/templates/post.html

```diff
@@ -23,52 +27,91 @@
                 <a href="javascript:quote('{{post.no}}');" title="Reply to this post">{{post.no}}</a>
             </span>
         </div>
+		{% if post.resto == 0 and post.filename %}
+        <div class="file" id="f{{post.no}}">
+            <div class="fileText" id="fT{{post.no}}">
+                File:
+                <a href="{{ thumb_uri.format(board_shortname=post.board_shortname) }}/{{ (post.asagi_preview_filename|string)[0:4] }}/{{ (post.asagi_preview_filename|string)[4:6] }}/{{ post.asagi_preview_filename}}">{{post.filename}}.{{post.ext}}</a>
+                (<span title="{{post.md5}}">{% if post.spoiler %}Spoiler, {% endif %}{% if post.fsize >= 1048576 %}{{(post.fsize / 1024 / 1024 )| round(2, 'floor')}} MB, {{post.w}}x{{post.h}}){% else %}{{(post.fsize / 1024 )| round(1, 'floor')}} KB, {{post.w}}x{{post.h}}){% endif %}</span>
+				{% if search_result %}
+					Replies: {{post.replies}} Files: {{post.images}} {% if post.posters %}Posters: {{post.posters}}{% endif %}
+				{% endif %}
+            </div>
+            <a class="fileThumb" href="{{ image_uri.format(board_shortname=post.board_shortname) }}/{{ (post.asagi_filename|string)[0:4] }}/{{ (post.asagi_filename|string)[4:6] }}/{{ post.asagi_filename}}" target="_blank">
+                <img src="{{ thumb_uri.format(board_shortname=post.board_shortname) }}/{{ (post.asagi_preview_filename|string)[0:4] }}/{{ (post.asagi_preview_filename|string)[4:6] }}/{{ post.asagi_preview_filename}}" data-md5="{{post.md5}}" style="height: {{post.tn_h}}px; width: {{post.tn_w}}px;" class="opImg postImg{% if post.spoiler %} spoiler{% endif %}" loading="lazy">
+            </a>
+        </div>
+        {% endif %}
         <div class="postInfo desktop" id="pi{{post.no}}">
-            {% if search_result %}
```

Comment:
You might know this, and even be using it (it's hard to tell in this GitHub view), but I'll mention there is a Python library called djhtml (`pip install djhtml`) and it will format jinja template when you run `djhtml .` in the templates directory.